Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN rustup default nightly
 RUN rustup target add wasm32-unknown-unknown --toolchain nightly
 
 #compiler ENV
-ENV CC clang
+ENV CC gcc
```

Comment:
Any reason for this change?

Reply:
It was reverted, originally I changed `gcc` -> `clang` because of an error like https://github.com/bytecodealliance/wasi/issues/14 which happens when building new dependencies (`polkadot-v0.9.28`).